### PR TITLE
fixes #15041 updated documentation

### LINKED ISSUE #15041: Logo Link URL
### Describe the bug
I changed the logo link URL to github.com and applied it. But when I clicked on the logo it redirected to index.php 

### To Reproduce

1. Go to 'settings'
2. Click on 'navigation panel'
3. Click on 'navigation panel'
4. Change the logo link URL to some other 
5. Click on logo.
6. It will redirect to index.php

### Expected behavior

It should have given some exceptions or warnings or should have redirected to the new URL. 

### Screenshots
Default index.php:

image

Changes to github.com:

image

Still index.php:

image

### Server configuration

- Operating system: Windows
- Web server: XAMPP
- PHP version: 7.3.0
- phpMyAdmin version: 4.8.5

### Client configuration

- Browser: Firefox
- Operating system: Windows

### PR DESCRIPTION
**Pull request by ravgeetdhillon**
### Description

I have added the updated documentation for Navigation Logo Link's configuration in the section: Navigational Panel Setup.

Fixes #15041 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Review by williamdes — APPROVED** [review]

File: doc/config.rst [hunk 1]
```diff
@@ -2006,7 +2006,8 @@ Navigation panel setup
 
     Enter :term:`URL` where logo in the navigation panel will point to.
     For use especially with self made theme which changes this.
-    For external URLs, you should include URL scheme as well.
+    For relative/internal URLs, you need to have leading ``./`` or trailing characters ``?``.
```
Comment (ibennetch) [inline doc/config.rst]: This is pretty good, but I think I'd change this line a bit:

> leading \`\`./\`\` and trailing \`\`?\`\`, such as \`\`./sql.php?\`\`.

Does that sound reasonable?

Comment (ibennetch) [inline doc/config.rst]: This is pretty good, but I think I'd change this line a bit:

> leading \`\`./\`\` and trailing \`\`?\`\`, such as \`\`./sql.php?\`\`.

Does that sound reasonable?

**Review by ibennetch — CHANGES_REQUESTED** [review]

File: doc/config.rst (hunk 1 above)
Comment (ravgeetdhillon) [reply doc/config.rst]: Yea. It would be better to give it as an example.

**Review by williamdes — APPROVED** [review]

**Comment by williamdes** [comment]
@ravgeetdhillon Can you please post a comment with your [signoff](https://github.com/phpmyadmin/phpmyadmin/pull/15124/checks?check_run_id=101302646) line ?

**Comment by ravgeetdhillon** [comment]
Signed-off-by: Ravgeet Dhillon <ravgeetdhillon@gmail.com>

**Comment by williamdes** [comment]
Merged, thanks @ravgeetdhillon !

**Pull request merged**